run_scripts/demog_phone_to_uuid.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import json
+import os
+import time
+
+from core_data_modules.traced_data import TracedData, Metadata
+from core_data_modules.traced_data.io import TracedDataCSVIO, TracedDataJsonIO
+from core_data_modules.util import PhoneNumberUuidTable
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
```

Comment:
You can put in a help string here that tells me what this program does

Reply:
I think you run this then the generate_outputs stage?

Is there any reason to do this as a separate stage rather than within the generate_outputs stage?

Reply:
I wanted to keep it separate so that we can do away early, with using the original dataset with phone numbers which won't change and any subsequent runs will only need to deal with the deanonymised dataset.